kubectl-plugin/pkg/cmd/job/job_submit.go

```diff
@@ -226,6 +226,11 @@ func (options *SubmitJobOptions) Validate() error {
 		if submissionMode != rayv1.InteractiveMode {
 			return fmt.Errorf("Submission mode of the Ray Job must be set to 'InteractiveMode'")
 		}
+		// BackoffLimit cannot co-exist with InteractiveMode
```

Comment:
minor: this reads a little better, I think. Same for other instances below of co-exist.

```suggestion
		// BackoffLimit is incompatible with InteractiveMode
```